### ISSUE TITLE
Important comment is repeated thrice in main checka11y.css file

### ISSUE THREAD

**Issue by jackdomleo7** [jackdomleo7/Checka11y.css#119]
### Details

The below comment should only appear in the `checka11y.css` file once, but it appears 3 times. _(NOTE: Version may be different to the time this was raised)_

```css
/*! Checka11y.css v1.3.3 | MIT License | github.com/jackdomleo7/Checka11y.css */
```

### Screenshot(s)

N/A

### What is Expected?
Important comment to only appear once in `checka11y.css`.

### What is actually happening?
Important comment is appearing 3 times in `checka11y.css`.

**Comment by caseynv** [comment]
Can i be asigned this for an outreachy contribution. Thanks

**Comment by tannerdolby** [comment]
Since caseynv was here first, they deserve first dibs. Otherwise, iff this becomes stale, I'm happy to be the "fallback" assignee!

**Comment by jackdomleo7** [comment]
Of course @caseynv ! Thank you.

**Comment by jackdomleo7** [comment]
Hi @caseynv, have you made any progress on this? If not, since it has been inactive for 3 months, it'll have to be unassigned and available for someone else to pick up (maybe @tannerdolby if he's still interested).

**Comment by tannerdolby** [comment]
👋  @jackdomleo7

Depending on @caseynv's response, I'm happy to take this on. It bugged me quite that I couldn't find the issue when we first discussed it awhile back, so I'm eager to get to the bottom of the problem.

**Comment by Mohit-Aasirwal** [comment]
Hey! Can I be assigned to this issue?

**Comment by jackdomleo7** [comment]
Hi @Mohit-Aasirwal, sure, thank you!

**Comment by jackdomleo7** [comment]
Hey @Mohit-Aasirwal, did you get anywhere with this?

**Comment by tannerdolby** [comment]
@jackdomleo7 👋 If no one responds or is assigned soon, I've got some free time this weekend to track down a fix for this. Hope you've been well mate!

**Comment by jackdomleo7** [comment]
@tannerdolby Absolutely! Considering we've not heard back from @Mohit-Aasirwal in 4 months, I'd be happy to assign it to you 🙂

**Comment by tannerdolby** [comment]
@jackdomleo7 Cool! Thanks :) You will probably laugh when you see the fix but the comment appearing thrice was expected based on the order of compiling for the partial files in `checka11y.scss`. I submitted a fix and now the comment only appears once at the top of `checka11y.css`.
